packages/node_modules/@ciscospark/widget-space/README.md

```diff
@@ -68,64 +71,52 @@ For the latest builds that are pulled from the head of the master branch:
 
 ### Build from Source
 
-1.  Follow these instructions to checkout and build the `react-ciscospark` repo <https://github.com/ciscospark/react-ciscospark/blob/master/README.md>
+1. Follow these instructions to checkout and build the `react-ciscospark` repo <https://github.com/ciscospark/react-ciscospark/blob/master/README.md>
 
-1.  To build the Space Widget, run the following from the root directory:
+1. To build the Space Widget, run the following from the root directory:
 
   ```sh
   npm run build:package widget-space
   ```
 
-
 ## Usage
 
 ### Quick Start
 
 If you would just like to get running immediately, follow these instructions to get a webpack-dev-server running with the widget.
 
-1.  Create a `.env` file in the root of the React project with the following lines, replacing the Xs with the appropriate value:
+1. Create a `.env` file in the root of the React project with the following lines, replacing the Xs with the appropriate value:
 
-    ```
+    ```txt
     CISCOSPARK_ACCESS_TOKEN=AN_ACCESS_TOKEN
     SPACE_ID=XXXXXXXXXXXXXXX
     TO_PERSON_EMAIL=XXXXX@XXXXXXXXX
     ```
-1.  From the root directory run: `npm run start:package widget-space`
+1. From the root directory run: `npm run start:package widget-space`
 
-### HTML
+### Configuration
 
-The easiest way to get the Spark Space Widget into your web site is to add the built resources and attach data attributes to your a container.
+When loading the widgets there are some configuration options you can provide:
 
-If you're using our CDN, skip to the next section.
-  -  Copy the resources in the `dist` directory to own project.
-  -  Add a `<script />` tag to your page to include the `bundle.js`
-  -  Add a `<link />` tag to include `main.css`
+| Name | Data API | Req? | Description |
+|:--|:-------|:-:|---|---|
+| `accessToken` | `data-access-token` | Y | Access token for the user account initiating the messaging session. <br>For testing purposes you can use a developer access token from <https://developers.ciscospark.com>. |
+| `initialActivity` | `data-initial-activity` |   | (default: `message`) Activity view to open with the widget. Available options: <ul><li>`message`: Message view</li><li>`meet`: Meet view (_Only available in 1 on 1 spaces_)</li></ul>|
+| `startCall` | `data-start-call` |   | (default: `false`) When present, widget will start in Meet view and initiate a call with the toPerson immediately. (_Only available in 1 on 1 spaces_) |
+| `logLevel` | `data-log-level` |   | (default: `silent`) When present, widget will log debug information to console. This can be set to: `error`, `warn`, `debug`, `info`, `trace`, or `silent` |
+| <td colspan=4>**Include only one of the following attributes**</td> |
+| `spaceId` | `data-space-id` |   | ID of the space you want to open. |
+| `toPersonEmail` | `data-to-person-email` |   | Email of the message recipient |
+| `toPersonId` | `data-to-person-id` |   | User Id of the message recipient |
```

Comment:
This table doesn't render. Changing the header separator to this fixes it:
`|:--|:-------|:-:|---|`

Also, I can't get the `colspan` to render properly. Perhaps this should become an HTML table, or the three exclusive attributes should move into a separate table.